app/assets/javascripts/admin/payments.js

```diff
@@ -71,12 +71,10 @@ function getCheckoutTransactions (button) {
     // Bind json data to copy button
     $("#copy_transactions button").attr("data-clipboard-text", data);
 
-    data = JSON.parse(data);
-
```

Comment:
probably related to this: the 'copy checkout transactions' functionality is broken